### PR TITLE
Automatically add -DPROFILER

### PR DESCRIPTION
**Pull request by pbartholomew08**

### CODE REVIEW

**Review by mathrack — CHANGES_REQUESTED** [review]
It looks good for merging. Please update the line 
``PROFILER = none# none, caliper (add -DPROFILER to DEFS if not none)``
to remove the comment about -DPROFILER. Please also update the README to remove the required update of the variables DEFS.

File: Makefile [hunk 1]
```diff
@@ -61,6 +61,10 @@ else ifeq ($(FFT),cufft)
 endif
 
 ### Add the profiler if needed
+ifeq ($(PROFILER),none)
```
Comment (mathrack) [inline Makefile:64]: ifneq ?

Comment (mathrack) [inline Makefile:64]: ifneq ?

**Review by mathrack — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (pbartholomew08) [reply Makefile:64]: ah, wasn't sure that existed

**Review by mathrack — APPROVED** [review]

**Pull request merged**